### ISSUE TITLE
Defining custom build directory for CSS outputs blank CSS file

### ISSUE THREAD

**Issue by alroman** [ucla-oarc-web/WebBlocks#301]
When defining a custom build directory for CSS, the output `blocks.css` and `blocks-ie.css` files are blank.  Defining custom directory like so:

``` ruby
WebBlocks.config[:build][:css][:dir] = 'style'
```

Note that this is not the case if I rename the custom build directory to the default 'css'

``` ruby
WebBlocks.config[:build][:css][:dir] = 'css'
```

**Comment by ebollens** [comment]
This is a duplicate of #274

**Comment by ebollens** [comment]
This should be resolved per #274 in the `master` branch.
